collect_app/src/main/res/layout/google_drive_deprecation_banner.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/google_drive_deprecation_banner"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:visibility="gone"
+    app:shapeAppearance="?shapeAppearanceLargeComponent"
+    tools:visibility="visible">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/margin_standard">
+
+        <TextView
+            android:id="@+id/message"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingStart="@dimen/margin_standard"
+            android:paddingEnd="@dimen/margin_standard"
+            android:text="@string/google_drive_deprecation_message"
+            android:textAppearance="?textAppearanceBody2"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/dismiss_button"
+            style="@style/Widget.MaterialComponents.Button.TextButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_small"
+            android:text="@string/dismiss_button_text"
+            android:textAllCaps="true"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/learn_more_button"
+            app:layout_constraintTop_toBottomOf="@+id/message" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/learn_more_button"
+            style="@style/Widget.MaterialComponents.Button.TextButton"
```

Comment:
I think we should use `Widget.Material3.Button.TextButton` for these buttons and drop the `textAllCaps`.

Reply:
Drop the `textAllCaps` because you think they should not be capitalized? In https://m2.material.io/components/banners it is capitalized.

Reply:
Ah true. I'd forgotten that the "Banner" is an actual Material 2 component.